### ISSUE TITLE
Refresh relative times on monitor map location labels

### ISSUE THREAD

**Issue by justinkambic** [elastic/uptime#138]
**Is your feature request related to a problem? Please describe.**
We've discussed that in the monitor page, we should continuously update the values shown in the relative timestamps per location on the map component, regardless of whether the app is set to auto-refresh or not.

![image](https://user-images.githubusercontent.com/18429259/72281078-be15eb80-3607-11ea-97c1-8f18eacad3b0.png)

**Describe the solution you'd like**
The originally-proposed solution was to continuously update these values by some default interval.

**Describe alternatives you've considered**
An alternative approach would be to notify the user that the timestamps that they're viewing are stale, and they should refresh to see the latest data, perhaps with a UI link that will perform the refresh for them.

**Additional context**
Today, these values only refresh when the entire page fetches its data.

**Comment by alvarolobato** [comment]
@Justin-Kambic  does this actually need design? I see the screen with the lastest designs.

**Comment by justinkambic** [comment]
👍 no we didn’t need that label, thanks

**Comment by andrewvc** [comment]
I would appreciate @katrin-freihofner 's opinion on this however.

IMHO there's a balance between too many refreshes (e.g. once per second) which is distracting, and too few, which is confusing.

**Comment by katrin-freihofner** [comment]
@andrewvc that's true...but don't we state `a few seconds ago` until we reach a full minute? And would this mean the UI updates once a minute?

**Comment by andrewvc** [comment]
@katrin-freihofner I don't believe so, it should go transition to a specific number of seconds until it reaches a minute.

I was thinking a good refresh rate would be once every 5-10s or so.

@drewpost a penny for your thoughts?

**Comment by drewpost** [comment]
This touches on our concerns around this page having a time picker. As long as we have that time picker, we need to have everything on the page refresh at the same rate. Mixing refresh rates would be confusing and lead to potential misinterpretation. I think a simple solution that we've discussed to address this was to put the date/time stamp of the last refresh under the list of relative times. That way there's an anchor for the relative times. We can address a more dynamic refresh as part of our broader re think of the Monitors Overview page.

**Comment by justinkambic** [comment]
I think this discussion led to us deciding we should have an `updated @` visualization somewhere on the page that will denote how fresh/stale the data is, but I didn't see anything mentioning that on this thread.
